renovate.json

```diff
@@ -9,17 +9,20 @@
       "automergeType": "pr"
     },
     {
-      "depTypeList": ["ruby", "bundler", "Gemfile", "Gemfile.lock"],
+      "matchDepTypes": ["ruby", "bundler", "Gemfile", "Gemfile.lock"],
```

Comment:
Renovate changed `depTypeList` to `matchDepTypes`! Both do the same thing, but `depTypeList` is deprecated